principles/fp-019-term-stability.md

```diff
@@ -36,21 +36,22 @@ Implementation
 Detailed procedures for obsoleting a term are described on the OBO Academy page [Obsoleting an Existing Ontology Term](https://oboacademy.github.io/obook/howto/obsolete-term/). 
 
 <i><b>To obsolete a term, the ontology developer</b></i> MUST:
-1) Mark the term as obsolete
-  - OWL format: Add an "owl:deprecated" annotation property with value of "true^xsd:boolean"
-  - OBO format: Add an "is_obsolete: true" tag
-2) Prepend the string "obsolete " (including the space) to the term label <i>and</i> the `editor preferred term` (IAO:0000111), if present
-  - NOTE: To be consistent with [Principle 12](https://obofoundry.org/principles/fp-012-naming-conventions.html) "Naming Conventions", the syntax/format MUST be precisely as given above (that is, the exact string as shown, lowercase and space included, with no other punctuation before or after). Thus, the following are disallowed: "Obsolete {label}", "obsolete_{label}", "OBSOLETE {label}" (and variations thereof).
-3) Remove all existing logical axioms from the term
-4) Remove or replace all usages of the term elsewhere in the ontology. For example, if an ontology has A part-of B, and B has been deprecated with replacement by C, then the corrected axiom would be A part-of C. Likewise, if A part-of B, and B part-of C, if B is deprecated, then any part-of axiom involving B MUST be removed; that is, by stating instead A part-of C.
+1. Mark the term as obsolete:
+   - OWL format: Add an "owl:deprecated" annotation property with value of "true^xsd:boolean"
+   - OBO format: Add an "is_obsolete: true" tag
+2. Prepend the string "obsolete " (including the space) to the term label <i>and</i> the `editor preferred term` (IAO:0000111), if present
+   - NOTE: To be consistent with [Principle 12](https://obofoundry.org/principles/fp-012-naming-conventions.html) "Naming Conventions", the syntax/format MUST be precisely as given above (that is, the exact string as shown, lowercase and space included, with no other punctuation before or after). Thus, the following are disallowed: "Obsolete {label}", "obsolete_{label}", "OBSOLETE {label}" (and variations thereof).
+3. Remove all existing logical axioms from the term
+4. Remove or replace all usages of the term elsewhere in the ontology. For example, if an ontology has A part-of B, and B has been deprecated with replacement by C, then the corrected axiom would be A part-of C. Likewise, if A part-of B, and B part-of C, if B is deprecated, then any part-of axiom involving B MUST be removed; that is, by stating instead A part-of C.
 
 It is not necessary (and not advisable) to delete the textual definition.
 
 <i><b>To obsolete a term, the ontology developer</b></i> SHOULD:
-1) Indicate any exact term replacement:
+
+5. Indicate any exact term replacement:
```

Comment:
This information is in a different section from the section above (what the developer SHOULD do vs what the developer MUST do), so the numbering should restart with '1'.

Reply:
Well, I found it more practical to have a complete count. But since you are the host, I'll revert it.

The parentheses (besides the newline) might be one factor that disturbs the MD-->HTML converter, so I'd rather leave the dot.

Reply:
```suggestion
1. Indicate any exact term replacement:
```